src/pages/get-eth.js

```diff
@@ -0,0 +1,435 @@
+        >
+          <Link to="/get-eth/#dex">Try a Dex</Link>
+        </StyledCard>
+        <StyledCard
+          emoji=":robot:"
+          title="Wallets"
+          description="Some wallets let you buy crypto with a debit/credit card, bank transfer or even apple pay. Geographical restrictions apply."
+        >
+          <Link to="/wallets">More on wallets</Link>
+        </StyledCard>
+      </CardContainer>
+      <InfoBanner>
+        <Emoji svg text=":wave:" />
+        <InfoCopy>New to ETH? Here's an overview to get you started.</InfoCopy>
+        <InfoLink to="/eth">What's ETH?</InfoLink>
+      </InfoBanner>
+      <GradientContainer id="country-picker">
+        <h2>What country do you live in?</h2>
+      </GradientContainer>
+      <TwoColumnContent id="dex">
+        <Column>
+          <h2>Decentralized exchanges</h2>
+          <p>
+            Decentralized exchanges are open marketplaces for ETH and other
+            tokens.
+          </p>
+          <p>
+            There are no geographical restrictions. If someone is selling what
+            you want and accepting a payment method you can provide, you’re good
+            to go. Some example payments include other tokens, paypal and even
+            in-person cash deliveries.
+          </p>
+          <p>You will need a wallet to use a DEX.</p>
+          <Button to="/wallets">Get a wallet</Button>
+        </Column>
+        <CardColumn>
+          <SingleCard
+            emoji=":gear:"
+            title="Localcryptos.com"
+            description="Accepts a wide range of payment types"
+          >
+            <Link to="https://localcryptos.com">Try Localcryptos</Link>
+          </SingleCard>
+        </CardColumn>
+      </TwoColumnContent>
+      <TwoColumnContent>
+        <Image fluid={data.image.childImageSharp.fluid} />
+        <Column>
+          <h2>Keeping your ETH safe</h2>
+          <p>
+            The supply of ETH isn’t controlled by any government or company - it
+            is decentralized. This is great because it means it’s open to
+            everyone.{" "}
+          </p>
+          <p>
+            This means you need to take the security of your funds seriously.
+            With ETH, you’re not trusting a bank to look after your money,
+            you’re trusting yourself.
+          </p>
+          <h3>Protect your ETH in a wallet</h3>
+          <p>
+            You should keep your ETH in a wallet you control, not an exchange.
+            This helps keep your funds safe because only you can access it. If
+            you leave it in an exchange account and that exchange is hacked, you
+            could lose your funds.{" "}
+          </p>
+          <h3>Your public address</h3>
+          <p>Your wallet should have a public address that looks like this:</p>
+          <CodeBox>
+            <Code>
+              0xAc03BB73b6a9e108530AFf4Df5077c2B3D481e5A0xAc03BB73b6a9e108530AFf4Df5077c2B3D481e5A
```

Comment:
Important distinction: this is not a public address. This looks like a public key.

The Ethereum (public) address is the first 20 bytes of the SHA3 hashed public key. 

More on that here: https://vomtom.at/ethereum-private-and-public-keys/
This is something we should explain the distinction of 😄 , e.g. on https://www.notion.so/efdn/Create-new-Ethereum-accounts-keys-page-bbf18d9f22d440439230667367cc352b

Reply:
I see you updated the description but we should use an example address (i.e. 42 chars w/ the `0x`)

Reply:
This is meant to be an example of a wallet ETH address, let's chat through what this example should be.